demo/api/src/app.module.ts

```diff
@@ -125,7 +128,12 @@ export class AppModule {
                     sitePreviewSecret: config.sitePreviewSecret,
                 }),
 
-                RedirectsModule.register({ customTargets: { news: NewsLinkBlock }, Scope: RedirectScope }),
+                RedirectsModule.register({
+                    imports: [MikroOrmModule.forFeature([News]), PredefinedPagesModule],
```

Comment:
Here the module imports the dependencies of the target URL service. This doesn't seem correct to me. @fraxachun you did this differently for the user permissions module, could you share some insights?

Reply:
See change here: https://github.com/vivid-planet/comet/pull/3624/files#diff-b35306ba1fd418ff77fbeaf05bd6c6546bdb3f0dd965b9c70bf5813cfb9df007R22.

Reply:
I tried this here: 5b9f995a20968e9b2fd7dd6abf3d94a52c337f23. Now it requires an additional module that exports the service, which doesn't seem right. Maybe we can discuss this today.